workflows/utilities/wf_organism_parameters.wdl

```diff
@@ -215,7 +215,7 @@ workflow organism_parameters {
     String rsv_b_nextclade_ds_tag = "2024-08-01--22-31-31Z"
     String rsv_b_nextclade_ds_name = "nextstrain/rsv/b/EPI_ISL_1653999"
     Int rsv_b_genome_len = 15500
-    String rsv_b_kraken_target_organism = "Human orthopneumovirus"  
+    String rsv_b_kraken_target_organism = "human respiratory syncytial virus"  
```

Comment:
Is "Human respiratory syncytial virus B" not present on the DB? why was this decision made?

Reply:
For some reason I have never gotten hits with "Human respiratory syncytial virus B"